### PR TITLE
Adding html and css for four-card exercise

### PR DESCRIPTION
**Pull request by papamaemae**
Is my html and css code okay to merge to main? 
All feedback comments appreciated :)

### CODE REVIEW

**Review by aishbhisikar — COMMENTED** [review]
Your desktop view looks good almost the same as the given screenshot which we have, but at the bottom, the scrollbar is coming which means it is not responsive, you should work on media queries.
your code is understandable, good use of naming conventions.

File: four-cards.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        </div>
+    </div>
+    <div class = "card calculator">
+        <h3>Calculator</h3>
+        <p class = "toolDescription">Uses data from past projects to provide better delivery estimates</p>
+        <img class="icon" src="icon-calculator.svg" alt="calculator icon">
+    </div>
+</main>
+
+</body>
```
Comment (ghost) [inline four-cards.html:43]: Closing html tag missing

File: four-cards.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <h2 class = "lightH2">Reliable, efficient delivery</h2>
+    <h2 class = "boldH2">Powered by Technology</h2>
+    <p class = "intro">Our Artificial Intelligence powered tools use millions of project data points to ensure that your project is successful</p>
+</header>
+
+<main class = "container">
+    <div class = "card supervisor">
+        <h3>Supervisor</h3>
+        <p class = "toolDescription">Monitors activity to identify project roadblocks</p>
+        <img class="icon" src="icon-supervisor.svg" alt="magnifying class icon">
```
Comment (ghost) [inline four-cards.html:22]: It's a good practice to keep all images in a one sub folder named images or img

File: four-cards.css [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    margin: 0;
+}
+.card {
+    height: 150px;
+    border-radius: 5px;
+    padding: 35px;
+    margin: 35px 35px 0 35px;
+    background-color: #FFFFFF;
+}
+.supervisor {
```
Comment (ghost) [inline four-cards.css:48]: It's nice to have box-shadow. It's in the design.

File: four-cards.css [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+}
+.karma {
+    border-top: #D8B92D solid 5px;
+}
+.calculator {
+    border-top: #3F93C4 solid 5px;
+    margin-bottom: 80px;
+}
+.icon {
+    float: right;
```
Comment (ghost) [inline four-cards.css:62]: That's nice! Easy solution. I never thought of it. I used flex to move image here.

File: four-cards.css [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+    line-height: 1.5;
+}
+.header{
+    background-color: #F7F1FD;
+    padding: 80px 35px 35px 35px;
+    margin: auto;
+    text-align: center;
+}
+.lightH2 {
+    font-weight: lighter;
```
Comment (ghost) [inline four-cards.css:13]: We will learn in week 5 how to use Google Fonts. So, it would be better to use values (eg. 500, 800) as font-weight from then.

**Review by ghost — COMMENTED** [review]
Excellent work! It works fine. Just at iPad view on my browser (Chrome), left and right cards get squeezed. So, I think it's better to increase min-width in media queries by keeping one column view for mobile and iPad.

File: four-cards.css (hunk 4 above)
Comment (papamaemae) [reply four-cards.css:62]: Thank you!

File: four-cards.css (hunk 3 above)
Comment (papamaemae) [reply four-cards.css:48]: Thanks, Aishwarya pointed this out too. I'll add it into my code :)

File: four-cards.html (hunk 2 above)
Comment (papamaemae) [reply four-cards.html:22]: Cool, I didn't know this, but it makes sense. Thanks Abul!

File: four-cards.html (hunk 1 above)
Comment (papamaemae) [reply four-cards.html:43]: Oops 🙃

**Comment by papamaemae** [comment]
@abulyousuf thanks very much for your really useful feedback. I'll make the changes you suggest 😄

**Comment by papamaemae** [comment]
@aishbhisikar thanks for your feedback Aishwarya, I definitely need to work on media queries more!

**Pull request merged**